ToDoList/ToDoList.API.Tests/Utilities/ControllerActionExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace ToDoList.API.Tests.Utilities
+{
+    internal static class ControllerActionExtensions
+    {
+        public static async Task<HttpResponseMessage> ExecuteAction<T>(this T controller, Func<T, Task<IHttpActionResult>> action) 
```

Comment:
I like it, I also like the fact that you pass `controller` into the `action` selector. Tests look really clear and everything, good stuff :)

Nitpicking note: consider giving the `action` a bit longer name - `actionSelector` (or `selectAnActionYouWantToTestPleasePleasePlease` (just making fun of myself, sorry 👼)) might guide the method's user even more directly…